app/backend/src/couchers/servicers/auth.py

```diff
@@ -342,7 +348,7 @@ def SignupFlow(
             user.profile_gallery_id = profile_gallery.id
 
             if flow.filled_feedback:
-                form = ContributorForm(
+                form_ = ContributorForm(
```

Comment:
Why underscore here?

Reply:
@kalvdans `form` is defined above, and has a different type: `form = request.feedback`